### PR TITLE
Payment QR code process

### PR DESCRIPTION
**Pull request by yuliaveres**

### CODE REVIEW

File: Token/Controllers/Settings/QRCodeController.swift [hunk 1]
```diff
@@ -20,6 +20,10 @@ import CoreImage
 class QRCodeController: UIViewController {
 
     static let addUsernameBasePath = "https://app.tokenbrowser.com/add/"
+    
+    static let addUserPath = "/add/"
+    static let paymentWithUsernamePath = "/pay/"
+    static let paymentWithAddressPath = "/pay/ethereum:"
```
Comment (faymontage) [inline Token/Controllers/Settings/QRCodeController.swift]: In our docs this is `ethereum:0x...` (no `/pay/` prefix) - but I've just done a review of the ecosystem and it seems like just bare address is a lot more common, with the rest being IBAN-style. I'll document this for you.

**Comment by faymontage** [comment]
yes! PaymentPresentable is a much-needed abstraction 😍

**Comment by roberthein** [comment]
Looking good 🚀 I'll have another look when it's finished.

File: Token/Protocols/PaymentPresentable.swift [hunk 2]
```diff
@@ -29,6 +29,12 @@ protocol PaymentPresentable: class {
 
 extension PaymentPresentable where Self: UIViewController {
     
+    /// Displays payment confirmation for given user info and parameters
+    ///
+    /// - Parameters:
+    ///   - userInfo:
+    ///   - parameters: include from, to, and value in wei fields
+
```
Comment (Elland) [inline Token/Protocols/PaymentPresentable.swift]: does this work with the extra newline? 🌀

**Comment by Elland** [comment]
Amazing stuff so far @yuliaveres 💪🏽 keep it up!

File: Token/Models/TokenUser.swift [hunk 3]
```diff
@@ -133,7 +133,7 @@ public class TokenUser: NSObject, NSCoding {
     }
     
     var userInfo: UserInfo {
-        return UserInfo(address: self.address, paymentAddress: self.paymentAddress, avatar: self.avatar, name: self.displayUsername, username: self.username, isLocal: true)
+        return UserInfo(address: self.address, paymentAddress: self.paymentAddress, avatar: self.avatar, avatarPath: self.avatarPath, name: self.displayUsername, username: self.username, isLocal: true)
```
Comment (Elland) [inline Token/Models/TokenUser.swift]: Forgot the name/username? 👍

Comment (yuliaveres) [inline Token/Models/TokenUser.swift]: yes 😄

Comment (yuliaveres) [inline Token/Models/TokenUser.swift]: but still wip 🙈

**Comment by Elland** [comment]
Looks good to me. Simple and to the point. 👍🏽

**Pull request merged**